scripts/deep_learning/hierarchy/LaterTier/TrainLaterTierModel_tracks.ipynb

```diff
@@ -0,0 +1,994 @@
+{
```

Comment:
My immediate reaction to this is how much of this is identical to the showers case? It looks like large chunks of it are the same. So there's probably some scope to refactor out common parts.

Reply:
I have been able to combine the later tier track and shower trainings. I have not been able to do the same for the primary training cases.